sql/payday.sql

```diff
@@ -55,7 +55,7 @@ CREATE TABLE payday_payments_done AS
 
 DROP TABLE IF EXISTS payday_payment_instructions;
 CREATE TABLE payday_payment_instructions AS
-    SELECT participant, team, amount
+    SELECT s.id, participant, team, amount, due
       FROM ( SELECT DISTINCT ON (participant, team) *
```

Comment:
As I understand it, ```ts_start``` is defined at the start of payday. This would make it 
difficult to incorporate this into the ```current_payment_instructions``` view as a condition that would be meaningful at other times as well as at payday.

I'd also like to understand the situation where ```mtime``` in ```payment_instructions``` 
could/would be >= payday ```ts_start```. e.g. can a user specify a start time that is in the future,
for a subscription? Was unable to find any such provision in the dev environment.